R/utils.R

```diff
@@ -66,14 +66,22 @@ path_dropbox_2dii <- function(...) {
 #' Before performing an operation on a file assumed to be found in a given
 #' directory, validate this file exists and give indicative error if not.
 #'
-#' @param path Character vector indicating the directory of a file
+#' @param path Character vector indicating the directory of a file.
 #'
-#' @return A boolean.
+#' @return NULL
 #'
 #' @export
 validate_file_exists <- function(path) {
   valid_file_path <- file.exists(file.path(path))
-  stopifnot(valid_file_path)
+
+  if (!valid_file_path) {
+    rlang::abort(c(
+      "Detected invalid file path.",
```

Comment:
Nit:

> If the cause of the problem is clear, use “must”:
-- https://style.tidyverse.org/error-messages.html#problem-statement

Suggested:

"`path` must point to an existing file."